core/templates/domain/exploration/ExplorationObjectFactorySpec.ts

```diff
@@ -143,6 +156,10 @@ describe('Exploration object factory', () => {
           default_outcome: {}
         }
       },
+      classifier_model_id: null,
+      card_is_checkpoint: false,
+      linked_skill_id: null,
+      next_content_id_index: Number(undefined)
```

Comment:
Looking at the codebase I think you can use just `1`

Reply:
Done.